CHANGES/494.feature

```diff
@@ -0,0 +1 @@
+Added support for cross repository blob mount.
```

Comment:
can you mention this in the push workflow?

Reply:
I have just added a paragraph that advertises support for the implemented feature.